### DISCUSSION TITLE
Facing error during Installation

### DISCUSSION THREAD

**Discussion by b1kr3m** [prasanthrangan/hyprdots#1328] [Q&A]
![error2](https://github.com/prasanthrangan/hyprdots/assets/97847882/449fe878-20f1-4791-8732-405413b8bc0a)
![Error](https://github.com/prasanthrangan/hyprdots/assets/97847882/56b8bb51-2373-4804-8a9d-f93ea870982c)
Could you please help me to how i can solve that problem........

**Comment by kRHYME7** [comment]
Hello can you do this ``` Hyde-install ``` ?

**Reply by b1kr3m** [reply]
No no, simply I just clone from the repo and execute the install.sh
